docker-compose.yml

```
@@ -1,10 +1,10 @@
-version: '3'
+version: '4'
```

Looks like we want to keep this as-is
https://docs.docker.com/compose/compose-file/compose-versioning/

```suggestion
version: '3'
```